src/Microsoft.AspNet.Razor/TagHelpers/TagHelperDescriptorProvider.cs

```diff
@@ -12,7 +12,7 @@ namespace Microsoft.AspNet.Razor.TagHelpers
     /// </summary>
     public class TagHelperDescriptorProvider
     {
-        private const string CatchAllDescriptorTarget = "*";
+        public static readonly string CatchAllDescriptorTarget = "*";
```

Comment:
just wondering: think it would make sense for this to be `public const string CatchAllTagName` in the `TargetElementAttribute` class?  that would make explicit use of that attribute a tad easier to read and write.  (attribute usage is about the only excuse for public constants that aren't `enum` fields)